### ISSUE TITLE
Possessives which should be split

### ISSUE THREAD

**Issue by AngledLuffa** [UniversalDependencies/UD_English-EWT#154]
A variety of possessives should be split into two tokens.  For example:

Dominion Posts
bislas
hancocks
Birdies
Bridies
Levi`s - weird punctuation, but either way, should be tokenized with a possessive?

**Comment by nschneid** [comment]
Birdies and Bridies are both names of businesses (not sure which was the correct spelling in this instance).

Bisla's (possessive) is a business as well, as is Hancock Fabrics. And the Dominion Post seems to be a newspaper.

The Levi`s sentence is hard to parse. I decided to treat it as plural rather than possessive (but marked as a typo because of the backtick).
